File: NotificationTimeOut_OC/NotificationTimeOut/ViewController.m
//
//  ViewController.m
//  NotificationTimeOut
//
//  Created by hh on 16/5/4.
//  Copyright © 2016年 pxj. All rights reserved.
//

#import "ViewController.h"
#import "PXJNSNotificationTimeOut.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [[PXJNSNotificationTimeOut instance] PostSeesionNotificationStart];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event{
    [[PXJNSNotificationTimeOut instance] PostSeesionNotificationStart];
}


@end

File: NotificationTimeOut_Swift/NotificationTimeOut/ViewController.swift
//
//  ViewController.swift
//  NotificationTimeOut
//
//  Created by hh on 16/5/3.
//  Copyright © 2016年 hh. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    override func touchesBegan(touches: Set<UITouch>, withEvent event: UIEvent?) {
        PXJNSNotificationTimeOut.instance.PostSessionNotificationStart()
    }


}

